### ISSUE TITLE
Streams returning HTTP/500

### ISSUE THREAD

**Issue by MaxwellDPS** [LuatixHQ/opencti#5603]
## Description

Was using the stream to perform a perfect sync to a new platform see #4936. It had been working fine, but is now all of a sudden giving HTTP 500  when trying to load any stream or any url like `/stream/*`

## Environment

1. OS (where OpenCTI server runs): CentOS Stream 9
2. OpenCTI version: 5.15.4
3. OpenCTI client: frontend
4. Other environment details: Clustered k8s deployment

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1.  N/A

## Expected Output

Stream SEE connection

## Actual Output

HTTP/500 - No other data

Log from platform container
```json
{
    "category": "APP",
    "error": {
        "context": {},
        "message": "ERR no such key",
        "name": "ReplyError",
        "stack": "ReplyError: ERR no such key\n    at parseError (/opt/opencti/build/node_modules/redis-parser/lib/parser.js:179:12)\n    at parseType (/opt/opencti/build/node_modules/redis-parser/lib/parser.js:302:14)"
    },
    "level": "error",
    "message": "Error executing live stream 996fadb5-4a4b-4f13-88de-4675577dc2ea",
    "timestamp": "2024-01-19T00:38:57.277Z",
    "version": "5.12.4"
}
```

## Additional information

Tried a full restart and redis flush
New streams dont work. 
Seems like loss of functionality was really fast <1min is what Im seeing on the RMQ sync metrics

**Comment by MaxwellDPS** [comment]
Not sure if its related but also seeing this around the same time

```json
{
    "category": "APP",
    "error": {
        "context": {},
        "message": "Cannot read properties of undefined (reading 'length')",
        "name": "TypeError",
        "stack": "TypeError: Cannot read properties of undefined (reading 'length')\n    at taxiiV21DataHandler (/opt/opencti/build/src/manager/ingestionManager.ts:233:20)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Promise.all (index 0)\n    at async Promise.all (index 1)\n    at ingestionHandler (/opt/opencti/build/src/manager/ingestionManager.ts:288:5)\n    at /opt/opencti/build/src/manager/ingestionManager.ts:307:9\n    at pyt.#runHandlerAndScheduleTimeout (/opt/opencti/build/node_modules/set-interval-async/dist/set-interval-async-timer.cjs:36:13)\n    at Timeout._onTimeout (/opt/opencti/build/node_modules/set-interval-async/dist/set-interval-async-timer.cjs:29:13)"
    },
    "level": "error",
    "message": "[OPENCTI-MODULE] Taxii ingestion execution error for <REMOVED>",
    "timestamp": "2024-01-19T00:42:21.112Z",
    "version": "5.12.4"
}
```

**Comment by nino-filigran** [comment]
@MaxwellDPS I see that you did not indicate any steps to reproduce: is it because you don't know how to reproduce?

**Comment by richard-julien** [comment]
"ERR no such key" is an error that occurs when the stream is deleted from the redis. This situation must not happens as flushing directly the redis is not an authorized operation.

"Cannot read properties of undefined (reading 'length')", i think its already fixed in newest version of the platform

**Comment by MaxwellDPS** [comment]
Hey @richard-julien & @nino-filigran  thanks for the reply! 

@nino-filigran Yeah, there seems to be an issue with the datastore on this instance (#4936) But this was a unexpected Loss in cabin pressure scenario, so not sure a great way to recreate. 

@richard-julien On the Redis issue, this has largely resolved itsself after ~50m. To clarify here, redis was flushed following the initial stall.

Sequence of events
1. Stream Sync RMQ rate drops [Remote Not the problem instance]
2. On the remote instance all the streams returned HTTP/500s
3. Once I saw that + the Redis log I shut down the platform pods
4. I then flushed Redis and issued a restart of the Redis service
5. Once Redis was healthy I rescaled the platform pods and kubernetes rescheduled the workers
6. At this point the stream was still broken
